specs/engage-digital_openapi3.yaml

```diff
@@ -8214,6 +8236,9 @@ components:
           type: boolean
         error_message:
           type: string
+        hsm_template:
+          description: Only on WhatsApp Sources.

```

Comment:
to be completely honest I'm not sure why we displayed this in the API 😅 it will be most likely useless for customers but we never know it may help debugging at some point (different namespace in ED and in the business manager)